src/app/main/component/events/components/event-editor/components/create-event-dates/date-time/date-time.component.spec.ts

```
@@ -0,0 +1,135 @@
+      onlineLink: new FormControl(''),
+      place: new FormControl(''),
+      appliedLinkForAll: [false],
+      appliedPlaceForAll: [false]
+    });
+
+    component.dayNumber = 1;
+    component.daysForm = fb.array([dayFormGroup]);
+    component.dayFormGroup = dayFormGroup;
+
+    fixture.detectChanges();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)